stdlib/public/core/Codable.swift

```diff
@@ -938,7 +938,7 @@ public struct KeyedEncodingContainer<K: CodingKey> :
 ///
 /// Decoders should provide types conforming to `UnkeyedDecodingContainer` for
 /// their format.
-public protocol KeyedDecodingContainerProtocol {
+public protocol KeyedDecodingContainerProtocol<Key> {
```

Comment:
This was missed in the previous commit:

```suggestion
public protocol KeyedDecodingContainerProtocol {
```